src/drawer.py

```diff
@@ -103,7 +103,7 @@ def _draw_datapoint(self, m: Measurement):
 
     def _draw_vertical_axis_number(self, num_grid):
         num_data = self.trafo.grid_coord_to_data_label(num_grid, Axis.VERTICAL)
-        label = f'{num_data:.2e}'
+        label = f'{num_data:.2f}'
```

Comment:
I think, this isn't necessarily a good idea as the values can get very large or very small. The user should at least have the possibility to enable scientific notation.

Reply:
You're right. I reverted this change as this has nothing to do with adding CLI print output.